app/models/event.rb

```diff
@@ -1,14 +1,6 @@
 class Event < ApplicationRecord
-	#Allowing URL, end_date, and address2 to be empty 
-	validates :name, presence: true
-	validates :description, presence: true
-	validates :start_date, presence: true
-	validates :address1, presence: true
-	validates :city, presence: true
-	validates :state, presence: true, length: {is: 2} #Dropdown selection on front end?  
-	VALID_ZIP_REGEX= /\A\d{5}-?\d{4}?\z/
-	validates :zip, presence: true, format:{with:VALID_ZIP_REGEX}
-	validates :scholarship_available, inclusion: {in: [true, false]}
-	VALID_URL_REGEX = URI::regexp 
+	validates :name, :start_date, :source_updated, :source_type, :address1, :city, presence: true
+	validates :source_id, presence: true, uniqueness: {case_sensitive: false}
```

Comment:
Instead of validating for uniqueness, we would want to verify for a unique combination between the _id and _type.  Like this:  https://github.com/OperationCode/operationcode_backend/blob/master/app/models/git_hub_statistic.rb#L9

Reason being, we could in theory have the exact same id from another source (like facebook); however, we could not have two of the same id from the *same* source.